app/models/chat_channel.rb

```diff
@@ -139,68 +148,29 @@ def title_from_chatable
     end
   end
 
+  private
+
   def change_status(acting_user, target_status)
-    return if !ChatChannel.statuses.include?(target_status.to_s)
```

Comment:
This conditional is not necessary as we’re now controlling the `target_status` value.